addons/sale_purchase_stock/tests/test_unwanted_replenish_flow.py

```diff
@@ -16,17 +16,17 @@ def setUpClass(cls):
         cls.customer = cls.env['res.partner'].create(dict(name='Customer'))
 
         cls.supplier_A = cls.env['product.supplierinfo'].create({
-            'partner_id' : cls.vendor.id,
-            'min_qty' : 0.0,
-            'price' : 10.0,
-            'delay' : 0
+            'partner_id': cls.vendor.id,
```

Comment:
Please avoid useless diffs as such, it just messes with the history without a good reason :confused: 
Especially since you're not even changing a single thing in this file.